tests/unit/ui/components/TestAnimatedIcon.js

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import AnimatedIcon from 'ui/components/AnimatedIcon';
+import Icon from 'ui/components/Icon';
+
+describe(__filename, () => {
+  it('renders an Icon and sets CSS classes', () => {
+    const root = shallow(<AnimatedIcon name="foo" />);
```

Comment:
It may be overkill, but I think we tend to use consts in any case where we have a value in setup and in assertions, i.e., 

```js
const name = 'foo';
const root = shallow(<AnimatedIcon name={name} />);
expect(root).toHaveClassName(`AnimatedIcon-${name}`);
```

This comment applies to all the tests below.